packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -37,7 +37,7 @@ Expect error code 403 and error body:
 
 - Endpoint: `get /authentication/http/custom/valid`
 
-Expects header 'Authorization': 'SharedAccessKey valid-key'
```

Comment:
So is that correct, I didn’t see the original pr but aren’t headed supposed to be case insensitive and we should update the validation to relate that. Isn’t that Pascal case version the most common?

Reply:
They are same. Header name is case insensitive.

Reply:
Yeah so aren’t we being too restrictive here. (Aren’t some generator sending using Authorization) Wouldn’t the real fix just to make the validateHeader check be case insensitive

Reply:
Got it. Since the header name is case insensitive, it is not necessary to update the doc.